### DISCUSSION TITLE
programming in general

### DISCUSSION THREAD

**Discussion by octopuscaat** [BattlesnakeOfficial/feedback#165] [Q&A] [answered]
the starters dont really tell me how to crate more of my own or tell me how i am supposed to get my snake to do something, its more like giving you a bit of code and making you figure the rest out on your own. it doesent really explain what each line or bit of code does. its just there, and i dont get a lot of what they do. i am trying to loop a bit of code but it doesent work. without it it runs the code once and kinda gives up. i think it would be good to add a bit of extra help, so that you can make just a simple snake.

**Comment by andrewdmaclean** [comment, marked as answer]
Hey @octopuscaat 👋 

Here is a great getting started tutorials that builds on the starter snake (for Python anyways)
https://www.youtube.com/watch?v=YnVZYIoY5zI

Hope this helps!

Cheers,
Andrew
